content/_data/events/2019-10-02-eskilstuna.adoc

```diff
@@ -0,0 +1,12 @@
+---
+title: "Eskilstuna Meetup: Jenkins Configuration as Code (a.k.a jcasc) and pipelines"
```

Comment:
Most other events have listed the event name here and then placed the topic inside the body of the event.  Would you consider the same here so that the title is used more consistently.
```suggestion
title: "Eskilstuna DevOps Meetup"
```

The CONTRIBUTING doc says:

> Note, this is not subject of the event, but the name. Example: "Seattle JAM", "Jenkins World 2017". Basically, take a look at the events list on https://jenkins.io/events/ as though you were trying to choose events you would go to. "August JAM" is not specific enough, but "Jenkins World 2017" is.